SWLOR.Game.Server/Data/Migrations/2018-10-25.5.sql

```diff
@@ -0,0 +1,321 @@
+		 1 , -- IsActive - bit
+		 N'DarkSide.ForceLeech' ,  -- ScriptName
+		 8 ,    -- BaseFPCost
+		 0 ,    -- BaseCastingTime 
+		 N'Inflicts your target with energy to sap their life, bringing a portion back to yourself.' ,  -- Description - nvarchar(1024)
+		 29 ,    -- PerkCategoryID
+		 6 ,    -- CooldownCategoryID
+		 3 ,    -- ExecutionTypeID
+		 NULL , -- ItemResref
+		 0 ,    -- IsTargetSelf Only
+		 10 ,    -- Enmity
+		 1 ,    -- EnmityAdjustmentRuleID
+		 NULL     -- CastAnimationID 
+	 )
+
+INSERT INTO dbo.PerkLevels (
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		1 , --Level
+		3 , --Price
+		N'Damage 2, Recovery 50%, 6 ticks'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels(
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		2 , --Level
+		4 , --Price
+		N'Damage 2, Recovery 50%, 9 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels ( 
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		3 , --Level
+		5 , --Price
+		N'Damage 4, Recovery 50%, 6 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels ( 
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		4 , --Level
+		6 , --Price
+		N'Damage 4, Recovery 75%, 6ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels ( 
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		5 , --Level
+		7 , --Price
+		N'Damage 4, Recovery 75%, 9 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels (
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		6 , --Level
+		8 , --Price
+		N'Damage 6, Recovery 75%, 9ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels (
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		7 , --Level
+		9 , --Price
+		N'Damage 8, Recovery 50%, 12 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels (
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		8 , --Level
+		10 , --Price
+		N'Damage 8, Recovery 75%, 12 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels (
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		9 , --Level
+		11 , --Price
+		N'Damage 12, Recovery 50%, 15 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevels(
+							PerkID ,
+							Level ,
+							Price , 
+							Description
+ )
+VALUES (
+		152 , --PerkID
+		10 , --Level
+		12 , --Price
+		N'Damage 12, Recovery 75%, 15 ticks.'  --Description
+    )
+
+INSERT INTO dbo.PerkLevelSkillRequirements (
+                            PerkLevelID ,
+                            SkillID ,
+                            RequiredRank                           
+ )
+
+VALUES (
+		2080 , --PerkLevelID
```

Comment:
This will be a problem because we can't guarantee PerkLevelID will always be 2080. If you look at the ServerFiles/Database/Scripts/AddPerkLevels.sql script you can see an example of how to use SCOPE_IDENTITY() to get the last inserted ID.

So the idea being:

1.) Insert the perk.
2.) Store SCOPE_IDENTITY() into a variable
3.) Insert the child record (PerkLevel) with that variable.
4.) Store SCOPE_IDENTITY() of the PerkLevel you just inserted
5.) Insert the child record (PerkLevelSkillRequirements) with that variable.

This stuff is tricky and will hopefully become easier when I've finished the tool editor.